crates/bevy_diagnostic/src/system_information_diagnostics_plugin.rs

```diff
@@ -93,9 +96,9 @@ pub mod internal {
 
     pub(super) fn setup_plugin(app: &mut App) {
         app.add_systems(Startup, setup_system)
-            .add_systems(First, launch_diagnostic_tasks)
+            .add_systems(First, launch_diagnostic_job)
```

Comment:
The `First` schedule runs first on every frame, it needs to be moved to the `Startup` schedule to have it run one time.